packages/finch-graphql-devtools/app/scripts/components/Code.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Box } from '@chakra-ui/react'
```

Comment:
while this isn't the extension or website, you didn't want to continue using material ui?

Reply:
I'm not a huge fan of material UI, plus wanted something that was not going to add another css-in-js solution.